src/numerics/IDA_Solver.cpp

```diff
@@ -446,23 +478,37 @@ void IDA_Solver::init(doublereal t0)
         #if CT_SUNDIALS_VERSION >= 30
             SUNLinSolFree((SUNLinearSolver) m_linsol);
             SUNMatDestroy((SUNMatrix) m_linsol_matrix);
-            #if CT_SUNDIALS_VERSION < 40
-                m_linsol_matrix = SUNBandMatrix(N, nu, nl, nu+nl);
-            #else
+            #if CT_SUNDIALS_VERSION >= 60
+                m_linsol_matrix = SUNBandMatrix(N, nu, nl, m_sundials_ctx.get());
+            #elif CT_SUNDIALS_VERSION >= 40
                 m_linsol_matrix = SUNBandMatrix(N, nu, nl);
+            #else
+                m_linsol_matrix = SUNBandMatrix(N, nu, nl, nu+nl);
             #endif
             if (m_linsol_matrix == nullptr) {
                 throw CanteraError("IDA_Solver::init",
                     "Unable to create SUNBandMatrix of size {} with bandwidths "
                     "{} and {}", N, nu, nl);
             }
-            #if CT_SUNDIALS_USE_LAPACK
-                m_linsol = SUNLapackBand(m_y, (SUNMatrix) m_linsol_matrix);
+            #if CT_SUNDIALS_VERSION >= 60
+                #if CT_SUNDIALS_USE_LAPACK
+                    m_linsol = SUNLinSol_LapackBand(m_y, (SUNMatrix) m_linsol_matrix,
+                                                    m_sundials_ctx.get());
+                #else
+                    m_linsol = SUNLinSol_Band(m_y, (SUNMatrix) m_linsol_matrix,
+                                              m_sundials_ctx.get());
+                #endif
+                    IDASetLinearSolver(m_ida_mem, (SUNLinearSolver) m_linsol,
+                                       (SUNMatrix) m_linsol_matrix);
```

Comment:
Indent level?

Reply:
What do you mean? That seems consistent with one of the ways we normally indent multiline statements.

Reply:
Oh, sorry, I was missing the context lines around this. I see now.